typescript/common-resolvers/src/types.ts

```diff
@@ -53,7 +58,15 @@ export type DbDataset = Omit<
   | "labelClasses"
   | "labelClassesAggregates"
   | "workspace"
->;
+> & { workspaceSlug: string };
+
+export type DbWorkspace = Omit<
+  GeneratedWorkspace,
+  "__export typename" | "type" | "datasets" | "memberships" | "plan"
```

Comment:
is there a key called "__export typename" ? 🤯

Reply:
Good catch! I'll change this

Reply:
Done in dcf831969de86dd10b1d5bcb12a4af1b5f5d5e33